### PR TITLE
Martingale Example - Support for the new alpaca data API

### PR DESCRIPTION
**Pull request by shlomikushchi**
udpating this example code to be able to choose which data source to use. 
originl examle used polygon second aggs
now we also enabled to use alpaca quote data 
to make it similar we don't act on every quote, unless 1 second past.

to change between data sources, change the global USE_POLYGON

### CODE REVIEW

File: examples/martingale.py [hunk 1]
```diff
@@ -84,40 +96,27 @@ async def handle_agg(conn, channel, data):
...
+        # Listen for quote data and perform trading logic
+        @conn.on(r'T\..+', [self.symbol])
+        async def handle_alpaca_aggs(conn, channel, data):
+            now = datetime.datetime.utcnow()
+            if now - self.last_trade_time < datetime.timedelta(seconds=1):
+                # don't react every tick unless at least 1 second past
+                return
+            self.last_trade_time = now
+            self.tick_index = (self.tick_index + 1) % (self.tick_size)
+            print(self.tick_index)
```
Comment (umitanuki) [inline examples/martingale.py:110]: Do you intend to keep this print? If so, I would suggest putting more readable message.

File: examples/martingale.py [hunk 2]
```diff
@@ -146,7 +145,45 @@ async def handle_trade(conn, channel, data):
             elif event_type != 'new':
                 print(f'Unexpected order event type {event_type} received')
 
-        conn.run([f'A.{self.symbol}', 'trade_updates'])
+        if USE_POLYGON:
+            conn.run([f'A.{self.symbol}', 'trade_updates'])
+        else:
+            conn.run([f'alpacadatav1/T.{self.symbol}', 'trade_updates'])
+
+    async def process_current_tick(self, tick_open, tick_close):
```
Comment (umitanuki) [inline examples/martingale.py:153]: Why is this async?

File: examples/martingale.py (hunk 1 above)
Comment (shlomikushchi) [reply examples/martingale.py:110]: deleted the print

File: examples/martingale.py (hunk 2 above)
Comment (shlomikushchi) [reply examples/martingale.py]: fixed

**Pull request merged**